### PR TITLE
Emotion - layout fixes following testing

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves visual bugs following emotion migration found by testers in https://docs.google.com/spreadsheets/d/1XcvUahodb2wLI1Jaji6VsuQbZ8Ld4cDeUZi5iJZ-v1Q/edit#gid=0

**Overall change:**

**MAPs** http://localhost:7080/nepali/media-23269034
- [x] Width of container not being correctly set as `width: 100%`
- [x] Added `2rem` padding-bottom of most-watched on page
- [x] Fix Headline padding
- [x] Fix Timestamp padding

**PGL** http://localhost:7080/japanese/features-and-analysis-42786589
- [x] Fix timestamp padding below 1024px

**STY** http://localhost:7080/mundo/23263889
- [x] Fix timestamp padding below 1024px
- [x] Fix author padding below 1024px
- [x] Fix margin below related content section
- [x] Onward journeys lose R/L padding below 400px - it should be .5rem

**Most watched** http://localhost:7080/nepali/media/video
- [x] L/R padding being added below 1024 px that shouldn't be https://i.imgur.com/fA3dr7D.png
- [x] ~Investigate tile size changes - is this just due to the number of items in the list~ no fix needed

**Radio schedules** http://localhost:7080/korean/bbc_korean_radio/liveradio / http://localhost:7080/pashto
- [x] ~L/R padding being added below 600px to schedules that shouldn't be  https://i.imgur.com/XT138FB.png~
**Psammead issue - needs fixing in `psammead-radio-schedule` https://github.com/bbc/psammead/issues/3963**

**On demand radio**
- [x] ~AMP - RTL investigate width weirdness. Investigated~ - no issue found

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams

**Testing:**

- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/MostWatched/index.jsx [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 import RelatedContentPromo from '../CpsRelatedContent/RelatedContentPromo';
 import RelatedContentPromoList from '../CpsRelatedContent/RelatedContentPromoList';
 
+const StyledCpsOnwardJourney = styled(CpsOnwardJourney)`
+  @media (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding: 0rem;
```
Comment (ryanmccombe) [inline src/app/containers/MostWatched/index.jsx]:
```suggestion
    padding: 0;
```

**Review by ryanmccombe — APPROVED** [review]

File: src/app/pages/MediaAssetPage/MediaAssetPage.jsx [hunk 2]
```diff
@@ -36,7 +39,9 @@ import { RequestContext } from '#contexts/RequestContext';
 import { GelPageGrid } from '#app/components/Grid';
 
 const StyledTimestamp = styled(Timestamp)`
-  padding-bottom: ${GEL_SPACING_DBL};
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
```
Comment (RichardPK) [inline src/app/pages/MediaAssetPage/MediaAssetPage.jsx:42]: This fix is the same one done for **PGL**s and **STYs** `StyledTimestamp` - they all restyle the `Timestamp` component in the same way. CodeClimate would flag it as repetition, but not sure it's worth tying these 3 components' styles to each other by pushing it into its own wee component.

File: src/app/containers/CpsRelatedContent/index.jsx [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
-
+import styled from '@emotion/styled';
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 import RelatedContentPromo from './RelatedContentPromo';
 import RelatedContentPromoList from './RelatedContentPromoList';
 
+const StyledCpsOnwardJourney = styled(CpsOnwardJourney)`
+  margin-bottom: 1rem;
```
Comment (simonsinclair) [inline src/app/containers/CpsRelatedContent/index.jsx]: Only thing, can we use a GEL_SPACING value for this? I think 1rem is equal to GEL_SPACING_DBL or GEL_SPACING_QUAD.

Comment (RichardPK) [reply src/app/containers/CpsRelatedContent/index.jsx]: Yea good shout! Will amend

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**